### PR TITLE
💬 Update about text and allow for future captions

### PR DESCRIPTION
**Pull request by chuckcarpenter**

### CODE REVIEW

File: src/components/About.astro [hunk 1]
```diff
@@ -16,14 +16,15 @@ const images = [
       <div class="flex flex-col">
         <div class="flex-grow">
           <h2 class="text-4xl text-dark-green lg:text-6xl">
-            Finally, premium coworking space in Middleburg
+            Finally, premium coworking space in the heart of historic Middleburg
```
Comment (RobbieTheWagner) [inline src/components/About.astro]:
```suggestion
            Premium coworking space in the heart of historic Middleburg
```

@chuckcarpenter I think this gets a little long for the heading, but I do like your suggestion. Perhaps we can remove `finally` to shorten it a tiny bit?

File: src/components/About.astro [hunk 2]
```diff
@@ -16,14 +16,15 @@ const images = [
...
           </h2>
 
           <p class="my-4 text-dark-green text-lg">
-            Don't drive all the way to Leesburg or Ashburn just for a nice
-            coworking space. 1787's Middleburg coworking space includes blazing
-            fast internet, unlimited snacks and drinks, and many membership
-            options to fit your needs and budget.
+            At 1787, you won't need to drive all the way to Leesburg or Ashburn 
+            for an elevated coworking space experience. With fiber-optic,
```
Comment (RobbieTheWagner) [inline src/components/About.astro]:
```suggestion
            for an elevated coworking space experience. With a fiber-optic, 
```

File: src/components/About.astro [hunk 3]
```diff
@@ -98,16 +99,17 @@ const images = [
       <div>
         <div class="swiper w-full xl:w-[500px] 2xl:w-[700px]">
           <div class="swiper-wrapper">
-            {images.map((image) => (
-              <div class="swiper-slide !h-72 !w-72 xl:!h-[525px] xl:!w-[525px]">
+            {images.map(({caption, src}) => (
+              <div class="swiper-slide !h-72 !w-72 xl:!h-[555px] xl:!w-[525px]">
```
Comment (RobbieTheWagner) [inline src/components/About.astro:103]: What are the implications of changing this height? Did you test out some sample captions and did this look good?

Comment (chuckcarpenter) [reply src/components/About.astro:103]: Yes, I did exactly that. They don't show at all unless it's taller. Any thoughts on potential issues?

Comment (RobbieTheWagner) [reply src/components/About.astro:103]: Should be fine for now 👍 . Mainly just worried about captions of different sizes on different screen sizes etc

**Review by RobbieTheWagner — APPROVED** [review]

**Pull request merged**